rstcheck.py

```diff
@@ -688,6 +688,13 @@ def visit_literal_block(self, node):
 
         raise docutils.nodes.SkipNode
 
+    def visit_paragraph(self, node):
+        """Check syntax of reStructuredText."""
+        find = re.search(r'\[[^\]]+\]\([^\)]+\)', str(node))
+        if find is not None:
+            raise self.document.reporter.warning(
```

Comment:
I don't think you need the `raise`. Calling the function itself should be [enough](https://github.com/docutils-mirror/docutils/blob/e88c5fb08d5cdfa8b4ac1020dd6f7177778d5990/docutils/writers/manpage.py#L682) to notify `docutils` of the warning.

It is a bit of a different style than the `raise docutils.nodes.SkipNode` in the other functions. That does raise an exception to communicate with `docutils`. But here, we are doing a function call so no raising of an exception is necessary.

Reply:
@myint Make sense, updated 👍